### ISSUE TITLE
Fill surface problem

### ISSUE THREAD

**Issue by zeex-pro** [mafanwei/libvlc#2]
Hello.

Thank you for your amazing work.

I have a problem with the player. If I change option to SURFACE_FILL my video does not fill the screen? I want to stretch video to fill the screen? Am I doing something wrong?

Thank you

**Comment by mafanwei** [comment]
Yes, this is a bug and I will fix it soon. Until then, you can use ```vlcVideoLibrary.getPlayer(). setScale(3f);``` to scale the video.
